Octokit.Reactive/Helpers/ConnectionExtensions.cs

```diff
@@ -28,6 +28,15 @@ public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connectio
             return GetPages(url, parameters, (pageUrl, pageParams) => connection.Get<List<T>>(pageUrl, pageParams, null).ToObservable());
         }
 
+        public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connection, Uri url, IDictionary<string, string> parameters, ApiOptions options)
```

Comment:
Actually, just make the body of `GetAndFlattenAllPages<T>(this IConnection connection, Uri url, ApiOptions options)` call into your new code:

```csharp
return connection.GetAndFlattenAllPages<T>(url, new Dictionary<string, string>(), options);
```

This should then leave you with the obsolete `GetPagesWithOptions<T>(Uri uri, ApiOptions options, Func<Uri, ApiOptions, IObservable<IApiResponse<List<T>>>> getPageFunc)` which can be safely deleted.